### PR TITLE
Release

### PR DESCRIPTION
**Pull request by RobdeMooij**
- Add readme
- Fix all code to run properly

### CODE REVIEW

**Comment by RobdeMooij** [comment]
Waiting on releases of `qcardia` packages and link to public paper

**Comment by RobdeMooij** [comment]
Should add MIT and Apache 2.0 (for DINO parts) licenses + DINO (significant) changes, see: https://opensource.stackexchange.com/questions/12263/what-type-of-notices-and-licenses-do-i-need-to-add-when-adding-and-modifying-c

~~Alternatively, I could rewrite the DINO utils to prevent all the multi/partial license troubles?~~

File: README.md [hunk 1]
```diff
@@ -1,2 +1,68 @@
...
+```
+data
+└── original_data
+    ├── MnM
+    │   ├── dataset
+    │   │   ├── A0S9V9
+    │   │   │   ├── A0S9V9_sa_gt.nii.gz
+    │   │   │   └── A0S9V9_sa.nii.gz
+    │   │   └── ...
+    │   └── dataset_information.csv
```
Comment (cianmscannell) [inline README.md]: original filename?

**Review by cianmscannell — APPROVED** [review]
At the end, of the README add a guideline for people to cite your work.

# Citation
If you find our work useful in your research please consider citing our paper:

@misc{demooij2024ssp,
      title={Self-supervised Pretraining for Cardiovascular Magnetic Resonance Cine Segmentation, 
      author={Rob A. J. de Mooij and Josien P. W. Pluim and Cian M. Scannell},
      year={2024},
      eprint={2409.18100},
      archivePrefix={arXiv},
      primaryClass={cs.CV},
      url={https://arxiv.org/abs/2409.18100}, 
}

**Pull request merged**